manifests/charts/default/templates/mutatingwebhook.yaml

```diff
@@ -113,6 +113,9 @@ webhooks:
       operator: DoesNotExist
     - key: istio.io/rev
       operator: DoesNotExist
+    - key: "kubernetes.io/metadata.name"
+      operator: "NotIn"
+      values: [ "kube-system" ]  
```

Comment:
Should we add all the namespaces that are ignored by Istio for injection?
https://github.com/istio/istio/blob/master/pkg/kube/inject/initializer.go#L38

Reply:
+1 - anytime these actually _are_ triggered they will be ignored immediately, so no point ever triggering them

Reply:
👍🏻